ci/install.sh

```diff
@@ -1,12 +0,0 @@
-set -euxo pipefail
```

Comment:
These scripts are used in Travis CI (which is not active I guess), so it would be nice to replace these with Github Actions CI

Reply:
I can try and add that, would you like to see it done in this PR?

Reply:
Thanks! I'd create a separate PR and leave the CI files intact in this PR.